tests/integration/minicluster_v2.go

```diff
@@ -76,7 +76,7 @@ func DefaultParams() map[string]string {
 		params.LocalStorageCfg.Path.Key:              path.Join("/tmp", testPath),
 		params.CommonCfg.StorageType.Key:             "local",
 		params.DataNodeCfg.MemoryForceSyncEnable.Key: "false", // local execution will print too many logs
-		params.CommonCfg.GracefulStopTimeout.Key:     "10",
+		params.CommonCfg.GracefulStopTimeout.Key:     "30",
```

Comment:
Too short query node graceful timeout.
Causing querynode_test's stop testing failure.
(search will return service not ready).

Reply:
only in integration test